pkg/ccl/changefeedccl/sink_kafka.go

```diff
@@ -441,9 +444,30 @@ func (c *saramaConfig) Apply(kafka *sarama.Config) error {
 		}
 		kafka.Version = parsedVersion
 	}
+	if c.RequiredAcks != "" {
+		val, err := parseRequiredAcks(c.RequiredAcks)
+		if err != nil {
+			return err
+		}
+		kafka.Producer.RequiredAcks = val
+	}
 	return nil
 }
 
+func parseRequiredAcks(a string) (sarama.RequiredAcks, error) {
+	switch a {
+	case "0", "NONE":
+		return sarama.NoResponse, nil
+	case "1", "ONE":
+		return sarama.WaitForLocal, nil
+	case "-1", "ALL":
+		return sarama.WaitForAll, nil
+	default:
+		return sarama.WaitForLocal,
+			fmt.Errorf(`invalid acks value "%s", must be "NONE", "ONE", or "ALL"`, a)
```

Comment:
nit: might want to explicitly mention that numbered values are also allowed here, i.e. say something like:
```
fmt.Errorf(`invalid acks value "%s", must be "NONE"/"0", "ONE"/"1", or "ALL"/"-1"`, a)
```